src/components/editor/editor-pane/tool-bar/table-picker/table-picker.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+
+
+.table-picker-container {
+  z-index: 1111;
+}
+
+body.dark {
+  div.table-cell {
+    @import "../../../../../style/variables.dark";
+    border-top: 1px solid $dark;
+    border-left: 1px solid $dark;
+  }
```

Comment:
Is there a specific reason why you bind all rules to divs?

Reply:
nope. removed